cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,181 @@
+/* eslint-disable cypress/no-force */
+/* eslint-disable cypress/unsafe-to-chain-command */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  // eslint-disable-next-line max-len
+  it('should have pagination with 5,10,20,25,50,100 values in selector and equal number of rows', () => {
+    // cy.get('.-pagination').should('have.select');
+
+    cy.get('[aria-label="rows per page"]')
```

Comment:
I don't see any reason to check all the combinations. Two would be enough

Reply:
Is three would be okay? Boundary and class partition.

Reply:
Done